include/jemalloc/internal/psset.h

```diff
@@ -0,0 +1,61 @@
+#ifndef JEMALLOC_INTERNAL_PSSET_H
```

Comment:
Could you write some design doc on how each of the APIs in this header is supposed to be used in the bigger picture of HPA?

Reply:
Shared a doc with you -- will probably let the design stabilize a bit before I want to describe things within a docs subdirectory.